frontend/src/App.js

```diff
@@ -1,32 +1,43 @@
+import React from "react";
 import {BrowserRouter, Switch, Route} from "react-router-dom";
 import LocationsPage from "./pages/LocationsPage";
 import Header from "./components/Header";
 import {ThemeProvider} from '@material-ui/core/styles';
 import {createMuiTheme} from '@material-ui/core/styles';
 import LocationDetailsPage from "./pages/LocationDetailsPage";
+import {geolocated} from "react-geolocated";
 
-function App() {
-    const darkTheme = createMuiTheme({
-        palette: {
-            mode: 'dark',
-        },
-    });
+class App extends React.Component{
```

Comment:
Why do you use a class component here? While they are not deprecated (and probably never will be due to so much legacy code) I strongly advice to use functional components for new code.